zmon_worker_monitor/builtins/plugins/http.py

```diff
@@ -157,6 +161,12 @@ def __init__(
         self.verify = verify
         self._headers = headers or {}
         self.oauth2 = oauth2
+        self.method = method.lower()
```

Comment:
this allows users to overwrite the method with arbitrary content, either make it have two underscores or move the check into `__request__`